server/liveblog/client_modules/client_modules.py

```diff
@@ -59,6 +63,53 @@ class ClientPostsService(PostsService):
     pass
 
 
+class ClientItemsResource(ItemsResource):
```

Comment:
We need Items resource because we follow the exact same structure as for posts. So, a post is a package which contains one ore multiple items. Here a comment is a package which contains an item. It doesn't make sense to post an empty package (that is why we need items)